readme.md

```diff
@@ -26,6 +26,11 @@ prettyBytes(1337);
 prettyBytes(100);
 //=> '100 B'
 
+// Include plus sign for positive numbers
+// (useful for displaying file size differences)
```

Comment:
This should be documented in the API section instead.

Reply:
You also need to mention the fact that `0` will have an empty space in front of it.

Reply:
fixed, thanks :)